Code/Tools/SceneAPI/SceneBuilder/Importers/AssImpBlendShapeImporter.cpp

```diff
@@ -40,7 +40,8 @@ namespace AZ
                 SerializeContext* serializeContext = azrtti_cast<SerializeContext*>(context);
                 if (serializeContext)
                 {
-                    serializeContext->Class<AssImpBlendShapeImporter, SceneCore::LoadingComponent>()->Version(3); // LYN-2576
+                    // Revision 4: Handle duplicate blend shape animations
+                    serializeContext->Class<AssImpBlendShapeImporter, SceneCore::LoadingComponent>()->Version(4); // LYN-2576
```

Comment:
// LYN-2576 is used again for a version bump?
Maybe the revision comment above could be added to this line instead?

Reply:
Moved it up, and replaced the ticket number with an actual comment. The Atom team originally started this by putting it on the line itself because it forces a conflict if two people change the same file at the same time. The gotcha I was running into there was short and not very useful version comments, plus it's harder in git than perforce to view the commit history to see what's changed each version. I'm experimenting with putting that version history here above the version change to see if that helps.